earthfile2llb/parser/EarthLexer.g4

```diff
@@ -39,7 +39,7 @@ NL: WS? COMMENT? CRLF;
 WS: [ \t] ([ \t] | LC)*;
 fragment CRLF: ('\r' | '\n' | '\r\n');
 fragment COMMENT: '#' (~[\r\n])*;
-fragment LC: '\\' [ \t]* CRLF;
+fragment LC: '\\' [ \t]* ([ \t] | CRLF | COMMENT)*;
```

Comment:
I did this within a line continuation because it makes the most sense in my head that these kind of rule should only apply within that scope